docs/user_guide/config.md

```diff
@@ -342,7 +347,9 @@ Below is an example implementation of a reverse proxy, using a custom Docker Com
 
           - Copy both files below into your `recipes` directory
           - In the `Caddyfile`, replace the placeholder domains with the custom domains your proxied services will use (see comments) 
-        
+          - If you prefer to host services on different subpaths instead of different subdomains, 
+          first update your `Caddyfile` (see the [Caddy docs](https://caddyserver.com/docs/caddyfile/matchers#path)) and then make sure to
+        update the base path variables in the `.env` file for each service (see the [`.env` docs](config.md#environment-variables) for more details.
```

Comment:
```suggestion
          - If you prefer to host services on different subpaths instead of different subdomains: 
              - Update your `Caddyfile` accordingly (see the [Caddy docs](https://caddyserver.com/docs/caddyfile/matchers#path))
              - Additionally, update the `*_BASE_PATH` variables in the `.env` file for the relevant services, e.g., `NB_NAPI_BASE_PATH` for the node API (see the [`.env` docs](config.md#environment-variables) for more details)
```